assets/css/style.css

```diff
@@ -162,6 +159,7 @@ label {
 
 .logo-placeholder img {
   margin: 10px 12px 10px 12px;
+  width: 100%;
 }
 
```

Comment:
- [ ] Please make sure you stick to the figma design. kindly arrange the padding and margins accordingly. and also try to apply the same fonts and typography.
## your design👇
![facest](https://user-images.githubusercontent.com/66992901/193131516-c50cf2d1-a39d-495d-93e6-f5c2ea4b2f7c.PNG)

## Figma design👇
![facefigma](https://user-images.githubusercontent.com/66992901/193131567-c642119b-a370-4784-bceb-868cf9764c88.PNG)
#### Please apply for all your cards😊.

Reply:
Hi @sajid-munawar it's done the second correction!!